web/src/components/molecules/WorkspaceSettings/DangerZone.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from "@emotion/styled";
+import { useCallback } from "react";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import Modal from "@reearth-cms/components/atoms/Modal";
+import { useT } from "@reearth-cms/i18n";
+
+export type Props = {
+  onWorkspaceDelete: () => Promise<void>;
+};
+
+const DangerZone: React.FC<Props> = ({ onWorkspaceDelete }) => {
+  const t = useT();
+  const { confirm } = Modal;
+
+  const handleWorkspaceDeleteConfirmation = useCallback(() => {
+    confirm({
+      title: t("Are you sure you want to delete this workspace?"),
+      icon: <Icon icon="exclamationCircle" />,
+      onOk() {
+        onWorkspaceDelete();
+      },
+    });
+  }, [confirm, onWorkspaceDelete, t]);
+
+  return (
+    <>
+      <Title>{t("Remove Workspace")}</Title>
+      <Text>
+        {t(
+          "Because at the current level of development, service A and service B are interdependent, and the workspace and personal accounts will be synchronized.",
+        )}
+        <br />
+        {t(
+          "So when you delete or create a workspace in service A and service B, they will all be synchronized with each other. Make sure you understand what you're doing.",
+        )}
+      </Text>
+
+      <Button onClick={handleWorkspaceDeleteConfirmation} type="primary" danger>
+        {t("Remove Workspace")}
+      </Button>
+    </>
```

Comment:
This is the old design. Please double check the figma file and update accordingly!

Reply:
@KaWaite , Red have updated the Figma to have this design for all settings pages.

Reply:
@issmail-basel Please check again. I talked with him the other day because the language here is confusing to figure out what he wanted to say, and it turned out there were two versions. The one you have here was an older version that he removed, so please check the file again